firewood/src/merkle.rs

```diff
@@ -1547,11 +2064,19 @@ mod tests {
 
     #[test_case(leaf(Vec::new(), Vec::new()) ; "empty leaf encoding")]
     #[test_case(leaf(vec![1, 2, 3], vec![4, 5]) ; "leaf encoding")]
-    #[test_case(branch(Some(b"value".to_vec()), vec![1, 2, 3].into()) ; "branch with chd")]
-    #[test_case(branch(Some(b"value".to_vec()), None); "branch without chd")]
-    #[test_case(branch(None, None); "branch without value and chd")]
+    #[test_case(branch(b"", b"value", vec![1, 2, 3].into()) ; "branch with chd")]
+    #[test_case(branch(b"", b"value", None); "branch without chd")]
+    #[test_case(branch_without_data(b"", None); "branch without value and chd")]
+    #[test_case(branch(b"", b"", None); "branch without path value or children")]
+    #[test_case(branch(b"", b"value", None) ; "branch with value")]
+    #[test_case(branch(&[2], b"", None); "branch with path")]
+    #[test_case(branch(b"", b"", vec![1, 2, 3].into()); "branch with children")]
+    #[test_case(branch(&[2], b"value", None); "branch with path and value")]
+    #[test_case(branch(b"", b"value", vec![1, 2, 3].into()); "branch with value and children")]
```

Comment:
Please use permalinks... the code changed and now I don't know what line you were talking about. 
Maybe [this one](https://github.com/ava-labs/firewood/pull/344/files#diff-6a6e300f3d9050248dbafa719531f14eabecc1bfac2169bb2527c81ee5474ecaR2071)?

Looks the same. This will get fixed with #512